src/modules/project/pages1/projectFunding/views/fundingSuccess/components/SuccessImageComponent.tsx

```diff
@@ -40,118 +51,186 @@ export const SuccessImageComponent = ({ currentBadge }: { currentBadge?: Badge }
+          <H3 color="neutral1.11" fontSize="3xl" regular>
             {t('Successfully contributed to')}
           </H3>
-          <H3 size={{ base: '2xl', md: '4xl', lg: '5xl' }} color={lightModeColors.primary1[12]} bold>
+          <H3 color="neutral1.12" bold fontSize="4xl">
             {project.title}
           </H3>
-          {currentBadge && (
-            <VStack w="full" spacing="0px">
-              <Image src={currentBadge.image} width="125px" />
-              <Body color="utils.whiteContrast" light>
-                {t('You won a Nostr badge!')}
-              </Body>
-            </VStack>
-          )}
         </VStack>
-      </HStack>
 
-      <HStack w="full" justifyContent="end">
-        <Tooltip w="100%" placement="top" label={copied ? t('copied') : t('copy')}>
+        <VStack spacing={2} w="full" zIndex={1}>
+          <Body color="neutral1.11" size="xl" regular pb={4} textAlign="center">
+            {t('Become an')}{' '}
+            <Tooltip label={t('Share your hero link to become a project ambassador')} placement="top">
+              <span style={{ position: 'relative', display: 'inline-block' }}>
+                <Body as="span" color="neutral1.12" textDecoration="underline dotted" display="inline">
+                  {t('Ambassador')}
+                </Body>
+              </span>
+            </Tooltip>{' '}
+            {t('for this project by spreading the word using your')}{' '}
+            <Tooltip label={t('A unique link that tracks contributions you helped generate')} placement="top">
+              <span style={{ position: 'relative', display: 'inline-block' }}>
+                <Body as="span" color="neutral1.12" textDecoration="underline dotted" display="inline">
+                  {t('Hero link')}
+                </Body>
+              </span>
+            </Tooltip>
+            {'. '}
+            {renderSharingStats()}
+          </Body>
+          {renderSignInPromptBody()}
+          <HStack
+            h="40px"
+            w="full"
+            p={2}
+            bg="whiteAlpha.700"
+            borderRadius={10}
+            border="1px solid"
+            borderColor="neutral1.7"
+            zIndex={1}
+          >
+            <Body color="neutral1.12" flex={1}>
+              <strong>{heroId ? t('Hero Link:') : ''}</strong> {heroLink.replace('https://', '')}
+            </Body>
+            <IconButton
+              aria-label={heroId ? 'Copy link' : 'Copy hero link'}
+              icon={<PiCopy />}
+              variant="ghost"
+              size="md"
+              onClick={handleCopy}
+            />
+          </HStack>
+        </VStack>
+        <HStack w="full" justifyContent="center" spacing={4} zIndex={1}>
+          <Button
+            size="lg"
+            variant="solid"
+            bg="whiteAlpha.800"
+            color="neutral1.11"
+            border="1px solid"
+            borderColor="neutral1.7"
+            borderRadius={8}
```

Comment:
unless it's a unique button, a variant and a colorScheme should be able to create all kinds of buttons in the design system.